src/features/DebugSession.ts

```diff
@@ -105,7 +114,7 @@ export class PickPSHostProcessFeature implements IFeature {
 
         if (this.waitingForClientToken) {
             this.clearWaitingToken();
-            return this.pickPSHostProcess();
```

Comment:
Yes, go ahead and merge it and see what you think.  Hopefully, you can figure out a better way to handle the PowerShell not initialized case.